ksqldb-engine/src/main/java/io/confluent/ksql/function/udf/map/MapUnion.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and limitations under the
+ * License.
+ */
+
+package io.confluent.ksql.function.udf.map;
+
+import io.confluent.ksql.function.udf.Udf;
+import io.confluent.ksql.function.udf.UdfDescription;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@UdfDescription(
+    name = "map_union",
+    description = "Returns a new map containing the union of all entries from both input maps. "
+        + "Returns NULL if all of the input maps are NULL.")
+public class MapUnion {
+
+  @Udf
+  public <T> Map<String, T> union(final Map<String, T> input1, final Map<String, T> input2) {
```

Comment:
Add `@UdfParameter` just so that it can compile without the `-parameters` in the pom file?

Reply:
makes sense